src/components/VideoItem.vue

```diff
@@ -148,6 +159,7 @@ export default {
         };
     },
     mounted() {
+        this.updateWatched([this.item]);
```

Comment:
Just wondering, why can't this be called in a batch? 

Like how we already do: https://github.com/search?q=repo%3ATeamPiped%2FPiped%20updateWatched&type=code

Reply:
Because you'd have to update them in in every component that uses it, as well as potential components that might use it in the future. There's nothing wrong with doing it like that but this way it's easier. Also, I think this way it doesn't make the db request until it's in view? I'm not exactly sure how Vue works to be honest

Reply:
But that method is already being called already in a few places, I would keep it that way to keep things fast :)

Is this line even necessary?

Reply:
All done :)